--- a/scripts/cmake/vcpkg_configure_meson.cmake
+++ b/scripts/cmake/vcpkg_configure_meson.cmake
@@ -220,7 +220,7 @@ function(vcpkg_internal_meson_generate_cross_file _additional_binaries) #https:/
             OUTPUT_VARIABLE MACHINE
             COMMAND_ERROR_IS_FATAL ANY)

-        if(MACHINE MATCHES "arm64")
+        if(MACHINE MATCHES "arm64|aarch64")
             set(BUILD_CPU_FAM aarch64)
             set(BUILD_CPU armv8)
         elseif(MACHINE MATCHES "x86_64|amd64")
