--- a/ports/libraw/portfile.cmake
+++ b/ports/libraw/portfile.cmake
@@ -1,23 +1,23 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO LibRaw/LibRaw
-    REF d4f05dd1b9b2d44c8f7e82043cbad3c724db2416
-    SHA512 5794521f535163afd7815ad005295301c5e0e2f8b2f34ef0a911d9dd1572c1f456b292777548203f9767957a55782b5bc9041c033190d25d1e9b4240d7df32b9
+    REF 52b2fc52e93a566e7e05eaa44cada58e3360b6ad
+    SHA512 f30ed1bd99df6d0759d9d820c586cd019a78cd7817a1a547565aeb6c53607c32ca19820e0aaf2f3270d4916abbaa892a70a27e6a6f71175fb226bb7d5bd22bf7
     HEAD_REF master
 )

 vcpkg_from_github(
     OUT_SOURCE_PATH LIBRAW_CMAKE_SOURCE_PATH
     REPO LibRaw/LibRaw-cmake
-    REF a71f3b83ee3dccd7be32f9a2f410df4d9bdbde0a
-    SHA512 607e6f76bcb57534da4f0c864b7a421f1ed49244468b1b52abe77f65aa599cae80715520b3a951294321b812deffd4f163757c9949f337571aa54f414ccc58a5
+    REF b82a1b0101b1e7264eb3113f1e6c1ba2372ebb7f
+    SHA512 b3f9807a902937db101c6e42b4275817420deed8774a05a68bca5a985cda688f27da3f473f55f2460af58bf1a2bf02578499e5401c8b7b677f46ca9f8f5faf9f
     HEAD_REF master
     PATCHES
-        findlibraw_debug_fix.patch
+    #findlibraw_debug_fix.patch
         lcms2_debug_fix.patch
         # Move the non-thread-safe library to manual-link. This is unfortunately needed
         # because otherwise libraries that build on top of libraw have to choose.
-        fix-install.patch
+       #        fix-install.patch
 )

