--- a/ports/mongoose/portfile.cmake
+++ b/ports/mongoose/portfile.cmake
@@ -5,8 +5,8 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO cesanta/mongoose
-    REF 8e520756366ca5739f13dc6ad65fcf269dbbc994 #v7.1
-    SHA512 a966a8b4e47e36da0f630c3cf343c85a1c1138508a82e506b21e4b8bd72573d0e0145318f97b32a67f423e033b348de76a00b780430e4e69d1a98bd7494a3e0a
+    REF 9173034b96cc312c3184f5d10324a0964132ab67 # master
+    SHA512 355095feb9b86f2df1d7942823892b24b761b0021731112ab69382c5decb2c22a67e77893dc09d5e7cf7bd2cd633433e8c10c667df8421802a23fc8e1bd7c1ac
     HEAD_REF master
 )
